src/utils/getFilteredExpenses.js

```diff
@@ -0,0 +1,26 @@
+function getFilteredExpenses(expenses, value) {
+  const filteredExpenses = expenses.filter((expense) => {
+    if (value.categories) {
+      return expense.category === value.categories;
+    }
+
+    if (value.userId) {
+      return expense.userId === Number(value.userId);
+    }
+
+    if (value.from && value.to) {
+      return (
+        new Date(expense.spentAt) > new Date(value.from) &&
+        new Date(expense.spentAt) < new Date(value.to)
+      );
+    }
```

Comment:
What if a user wants to get expenses that match a few parameters at the same time?